PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Data/CompatibilityProfileData.cs

```diff
@@ -22,6 +21,12 @@ public class CompatibilityProfileData : ICloneable
         [DataMember]
         public string Id { get; set; }
 
+        /// <summary>
+        /// The verison of the profile schema this profile object uses.
+        /// </summary>
+        [DataMember]
```

Comment:
I tried a few ways to set the automatic value of this to 1.0, but I couldn't work out how; I can't specify a version in the attribute.